test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] Kindly make sure that this file is actually ignored by git and is not present in your GitHub repository.

You should first untrack it by `git`, and then add it to the `.gitignore` file 😄 

To untrack the file, run` git rm --cached test.md` in your terminal